lib/flutter/flutter_launch.dart

```diff
@@ -70,7 +70,7 @@ class FlutterLaunchType extends LaunchType {
   }
 
   String getDefaultConfigText() {
-    return 'checked: true\ndebug: true\nroute:\n';
+    return 'checked: true\ndebug: true\nroute:\n# additional args for the flutter run command\nargs:\n';
```

Comment:
==> `# Additional args for the flutter run command:`

Also, we may want to break this into a multi-line string for readability (now that it's gotten a bit longer).